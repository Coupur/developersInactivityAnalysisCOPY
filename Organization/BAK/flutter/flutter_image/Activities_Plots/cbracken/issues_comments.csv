id,date,creator_login
330337567,2017-09-18 19:55:14,cbracken
330335367,2017-09-18 19:46:56,cbracken
