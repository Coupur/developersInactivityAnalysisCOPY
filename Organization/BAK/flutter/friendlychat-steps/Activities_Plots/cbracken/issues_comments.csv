id,date,creator_login
340799509,2017-10-31 15:29:09,cbracken
340799380,2017-10-31 15:28:45,cbracken
340799280,2017-10-31 15:28:26,cbracken
340799171,2017-10-31 15:28:08,cbracken
340799076,2017-10-31 15:27:51,cbracken
340798963,2017-10-31 15:27:33,cbracken
340798835,2017-10-31 15:27:09,cbracken
