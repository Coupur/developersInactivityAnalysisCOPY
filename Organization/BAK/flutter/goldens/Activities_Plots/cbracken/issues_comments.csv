id,date,creator_login
403948400,2018-07-10 20:04:09,cbracken
